test/contracts/contract_interface_polymorphism.aes

```diff
@@ -0,0 +1,9 @@
+contract interface II =
```

Comment:
Do we have a test if C does not implement f?

Reply:
Yes, [contract_polymorphism_missing_implementation](https://github.com/aeternity/aesophia/blob/8c42f69407778e946fd0ba2dff400bc941a2721b/test/contracts/contract_polymorphism_missing_implementation.aes) and it shows an error.